pkg/proxy/iptables/proxier.go

```diff
@@ -1625,11 +1625,13 @@ func (proxier *Proxier) syncProxyRules() {
 
 	// Finish housekeeping.
 	// TODO: these could be made more consistent.
+	klog.V(4).Infof("Deleting stale services IPs: %v", staleServices.UnsortedList())
```

Comment:
is this the right way to log this , %v?

Reply:
or is better to add a log entry by IP inside the for loop?

Reply:
"%v" seem to work fine for map[string]Empty{}